internal/db/migrations/011_delete_cascade.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE reservation_instances
+DROP CONSTRAINT reservation_instances_reservation_id_fkey,
+ADD CONSTRAINT reservation_instances_reservation_id_fkey
+FOREIGN key (reservation_id) REFERENCES reservations(id) ON DELETE CASCADE;
```

Comment:
This foreign key already has delete cascade in my database, so we might not need this at all.